src/screens/HtmlScreen.js

```diff
@@ -78,13 +78,18 @@ export const HtmlScreen = ({ navigation, route }) => {
       });
     }
 
-    // if there is no `param`, use the main `subQuery` values for `routeName` and a `webUrl`
+    const subParams = { ...(subQuery.params ?? {}) };
+    delete subParams.routeName;
```

Comment:
this is not actually needed.

when i wrote that, i was somehow under the impression that the `subQuery.routeName` would be part of the `subParams` unintentionally.

this might have been the case in an earlier version, but as it stands right now, that key should only be there, if explicitly given into the params, in which case it shoult probably stay there as well.